editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings/LPS/LPSListingSection.tsx

```diff
@@ -0,0 +1,116 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { useFormik } from "formik";
+import { useToast } from "hooks/useToast";
+import React from "react";
+import { Link } from "react-navi";
+import SettingsDescription from "ui/editor/SettingsDescription";
+import SettingsSection from "ui/editor/SettingsSection";
+import ErrorWrapper from "ui/shared/ErrorWrapper";
+import { Switch } from "ui/shared/Switch";
+
+import { useStore } from "../../../../lib/store";
+
+export const LPSListing = () => {
+  const [isListedOnLPS, setIsListedOnLPS, description, summary] = useStore(
+    (state) => [
+      state.isFlowListedOnLPS,
+      state.setIsFlowListedOnLPS,
+      state.flowDescription,
+      state.flowSummary,
+    ],
+  );
+
+  const toast = useToast();
+
+  const form = useFormik<{ isListedOnLPS: boolean }>({
+    initialValues: {
+      isListedOnLPS: isListedOnLPS ?? false,
+    },
+    validate: () => {
+      if (!description)
+        return {
+          isListedOnLPS:
+            "Service description required - please set via the 'About this flow' tab",
+        };
+      if (!summary)
+        return {
+          isListedOnLPS:
+            "Service summary required  - please set via the 'About this flow' tab",
+        };
+    },
+    onSubmit: async (values, { resetForm }) => {
+      const isSuccess = await setIsListedOnLPS(values.isListedOnLPS);
+      if (isSuccess) {
+        toast.success("Service settings updated successfully");
+        // Reset "dirty" status to disable Save & Reset buttons
+        resetForm({ values });
+      }
+    },
+  });
+
+  return (
+    <Box component="form" onSubmit={form.handleSubmit} mb={2}>
+      <SettingsSection>
+        <Typography variant="h2" component="h3" gutterBottom>
+          Listings
+        </Typography>
+        <Typography variant="body1">
+          Manage how this service is listed and indexed
+        </Typography>
```

Comment:
Very open to ideas here when it comes to the copy and how we present this to users.